assets/js/app.js

```diff
@@ -58,6 +66,31 @@ app.controller('AboutCtrl', ['$scope', '$rootScope', function($scope, $rootScope
   })
 }])
 
+app.controller('LoginCtrl', ['$scope', '$rootScope', function($scope, $rootScope) {
+  $rootScope.root = {
+    route: 'Login',  // this corresponds to the menu item that should be active
+    title: 'Login | GitSubmit'
+  }
+  app.controller('FormLoginCtrl', ['$scope', function($scope) {
+      $scope.submit = function() {
+        //TODO:Add code to handle the fields and send to approriate destination
+      };
+    }]);
+}])
+
+app.controller('SignupCtrl', ['$scope', '$rootScope', function($scope, $rootScope) {
+  $rootScope.root = {
+    route: 'Sign Up',  // this corresponds to the menu item that should be active
+    title: 'Sign Up | GitSubmit'
+  }
+
+    app.controller('FormSignupCtrl', ['$scope', function($scope) {
+      $scope.submit = function() {
+        //TODO:Add code to handle the fields and send to approriate destination
```

Comment:
I'd suggest adding a test method that shows the values entered e.g. `alert($scope.email + ', ' + $scope.username + ...);`